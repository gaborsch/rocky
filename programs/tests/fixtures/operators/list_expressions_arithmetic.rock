Let X be 1
Let Y be X with 2, 3, 4
Shout Y

Tommy says we gotta hold on
Let Tommy be with " to", " what", " we've", " got"
Shout Tommy

My life is always cold
Now is nothing
Build it up, up
Let my life be over now, now, now, now
Shout my life

The wolf is hungry, out on the street
Fear is the mind killer
Fury is the demon child
Hate is the only truth
Let the wolf be without fear, fury, and hate
Shout the wolf

Let Factorial be 6 times 5, 4, 3, 2, 1
Shout Factorial

Let chant be "Hey! " times 2, 2, 2
Shout chant

The songs are long forgotten,
Longing is exquisite,
Beauty is a memory,
Temptation is a lie
Let the songs be of beauty, and temptation, and longing
Whisper the songs 

(49 * 9 * 16 * 13 = 91728)

(operator precedence)
total is 2
let total be of 3 with 5     (total *= 3+5 )
shout it