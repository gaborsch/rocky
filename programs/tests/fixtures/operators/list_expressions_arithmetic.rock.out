10
we gotta hold on to what we've got
4
62190
720
Hey! Hey! Hey! Hey! Hey! Hey! Hey! Hey! 
91728
16
